web/client/components/map/openlayers/Map.jsx

```diff
@@ -85,7 +88,10 @@ class OpenlayersMap extends React.Component {
         resize: 0,
         registerHooks: true,
         hookRegister: mapUtils,
-        interactive: true
+        interactive: true,
+        onMouseOut: () => {},
+        mousePosition: false,
+        isMouseMoveOverTools: true
```

Comment:
Not needed anymore, I think you can remove it

Reply:
Done